flake.nix

```diff
@@ -21,6 +21,7 @@
           "ghc92"
           "ghc94"
           "ghc96"
+          "ghc98"
```

Comment:
I assume you were able to build/get a shell for this

Reply:
I will tell you in 18 hours. CI gives me some confidence at least

Reply:
I'm happy if it evaluates, everything else is gravy really